crates/gossip/proto/gossip.proto

```diff
@@ -16,8 +16,26 @@ message K2GossipMessage {
     // A gossip no diff protocol message.
     NO_DIFF = 3;
 
+    // A gossip disc sectors diff protocol message.
+    DISC_SECTORS_DIFF = 4;
```

Comment:
I like the organization of this file. Can you add a comment to the top of this enum (git won't let me put a suggestion there without a diff), something like:

```rust
/// These enum variants correspond to `K2Gossip*Message` types,
/// e.g. `INITIATE` -> `K2GossipInitiateMessage`.
```

Reply:
Good idea, done!